modules/virt-accessing-exported-vm-manifests.adoc

```diff
@@ -10,6 +10,7 @@ After you export a virtual machine (VM) or snapshot, you can get the `VirtualMac
 
 .Prerequisites
 
+* You have installed the OpenShift CLI (`oc`).
```

Comment:
```suggestion
* You have installed the {oc-first}.
```
There's a handy-dandy attribute for this now! Do you want to update these, or merge as-is?

Reply:
Oh, I didn't know. Please wait, I will update the PR to use the attribute. :)

Reply:
Hi @ShaunaDiaz, I updated the pull request and replaced all occurrences of "OpenShift CLI (\`oc\`)" with the attribute as you suggested. Once the pipelines finish running, unless there is a problem with previews you can go ahead and merge this. Thank you so much for your feedback!